src/components/Stepper/Step.tsx

```diff
@@ -139,19 +150,37 @@ export const Step: React.FC<StepProps> = ({
   completed,
   disabled,
   icon: Icon,
-  index,
+  index = 0,
+  numberOfSteps,
   onClick,
   subTitle,
   subTitlePillLabel,
   title,
   ...rootProps
 }) => {
   const classes = useStyles({});
+  const ariaLabel = `Step ${index + 1} of ${numberOfSteps}`;
```

Comment:
Should numberOfSteps be required? It seems like otherwise, this will read "Step X of undefined"

Reply:
The `numberOfSteps` prop is being passed in via the `Stepper` component when cloning the provided `Step`. We don't want to require this prop because then it would require all current implementations to pass it in. This is a prop that Chroma is controlling. Hopefully that makes sense.

Reply:
We do the same with the `index` prop.

Reply:
Does make sense -- thanks for the explanation.